### ISSUE TITLE
文件内硬编码调用 .cuda() 导致设备不可移植与潜在跨设备拷贝

### ISSUE THREAD

**Issue by H-Tenets** [LianpoWang/WLKDCA-Net#25]
在 forward_features 中存在 skip1 = x.cuda() 和 skip2 = x.cuda()（以及文件其他地方可能类似的直接 .cuda() 调用）。在模块内部直接把中间特征搬到 GPU 会引发多种问题：当模型被放到不同 device（CPU、特定 GPU 或多卡 DataParallel/Distributed）时会出现 device mismatch 或隐式跨设备拷贝；也会阻止灵活地把模型复制到其他设备或在推理时切换 device。正确做法是不要在模型前向中硬编码设备转换，而应让调用者负责把 model 和输入移动到目标设备，模块内部使用 x.to(x.device)／不显式调用 .cuda() 或者把需要持久化的张量注册为 buffer。建议修改为保存 skip1 = x.clone()（如果想保留副本）或直接 skip1 = x；若确实需要在前向内保持设备一致性，可使用 skip1 = x.detach() 而非 .cuda()，并在文档或断言中明确设备要求。
